src/components/FilterBar.jsx

```diff
@@ -1,112 +1,58 @@
 import React, { useState } from 'react';
-import PropTypes from 'prop-types';
-import {
-  Grid,
-  FormControl,
-  Select,
-  MenuItem,
-  Button,
-  Popover,
-} from '@mui/material';
-import AmountFilter from './AmountFilter';
-import GradeLevelFilter from './GradeLevelFilter';
+import { Menu, MenuItem, Toolbar, Button, Container } from '@mui/material';
 import useQueryParams from '../lib/useQueryParams';
 import sortOptions, { DEADLINE_ASC } from '../lib/sortOptions';
-import MajorFilter from './MajorFilter';
-import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+import TuneIcon from '@mui/icons-material/Tune';
+import PropTypes from 'prop-types';
+import ImportExportIcon from '@mui/icons-material/ImportExport';
 
-const FilterButton = ({ title, children, popoverWidth }) => {
+export default function FilterBar({ openFilter }) {
+  const [{ sortBy }, setQueryParam] = useQueryParams();
   const [anchorEl, setAnchorEl] = useState(null);
 
   return (
-    <>
-      <Button
-        variant="outlined"
-        onClick={(e) => setAnchorEl(e.currentTarget)}
-        endIcon={<ArrowDropDownIcon color="primary" />}
-        sx={{ m: 1, height: (theme) => theme.spacing(4) }}>
-        {title}
-      </Button>
-      <Popover
+    <Toolbar sx={{ bgcolor: 'primary.main' }}>
+      <Container
+        maxWidth="md"
+        sx={{ display: 'flex', justifyContent: 'space-between' }}>
```

Comment:
sadly once I surrouned the buttons with the `Container`, the `justifyContent` in the `Toolbar` didn't work

Reply:
also to keep the sort button on the left even when the filter button is hidden, I have to set the direction to `row-reverse` and put the sort button first

Reply:
nice!